src/main/java/calculator/Splitter.java

```diff
@@ -0,0 +1,17 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Splitter {
+    public static final  String DELIMITER = ",|:";
+    public static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
```

Comment:
습관적으로 optimize import와 formatting 단축키를 활용해주시는 것이 좋습니다. 
```suggestion
    public static final String DELIMITER = ",|:";
    public static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
```